query.go

```diff
@@ -197,6 +203,14 @@ func OptimisticReverseBatch() QueryOption {
 	}
 }
 
+// MaxBatchSize allows the caller to limit the number of items fetched
+// in a batch.
+func MaxBatchSize(maxSize uint) QueryOption {
```

Comment:
We should note that this is bound by `wire.MaxGetCFiltersReqRange`.

Reply:
> We should note that this is bound by `wire.MaxGetCFiltersReqRange`.

I actually avoided document that as it seems to me that the query options are designed to be used in different types of queries and not only in filters query.
That being said, I am not against adding a note in the spirit of "If this option is used in filters query it is bound to `wire.MaxGetCFiltersReqRange`".
LMK what you think.

Reply:
Ah I see, good point. Maybe just say that each batch is bound to its request's max size, but that may already be obvious? Not a blocker in any case.